nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -161,49 +161,13 @@ so even in a non-graphical session. If you prefer to configure the
 network manually, disable NetworkManager with
 `systemctl stop NetworkManager`.
 
-On the minimal installer, NetworkManager is not available, so
-configuration must be perfomed manually. To configure the wifi, first
-start wpa_supplicant with `sudo systemctl start wpa_supplicant`, then
-run `wpa_cli`. For most home networks, you need to type in the following
-commands:
-
-```plain
-> add_network
-0
-> set_network 0 ssid "myhomenetwork"
-OK
-> set_network 0 psk "mypassword"
-OK
-> set_network 0 key_mgmt WPA-PSK
-OK
-> enable_network 0
-OK
-```
-
-For enterprise networks, for example *eduroam*, instead do:
-
-```plain
-> add_network
-0
-> set_network 0 ssid "eduroam"
-OK
-> set_network 0 identity "myname@example.com"
-OK
-> set_network 0 password "mypassword"
-OK
-> set_network 0 key_mgmt WPA-EAP
-OK
-> enable_network 0
-OK
-```
-
-When successfully connected, you should see a line such as this one
-
-```plain
-<3>CTRL-EVENT-CONNECTED - Connection to 32:85:ab:ef:24:5c completed [id=0 id_str=]
-```
+On the minimal installer, the configuration must be perfomed on the
+command line. To configure the wifi, first start the NetworkManager
+with `sudo systemctl start NetworkManager`, then run
+`sudo nmtui connect` to guide you through the process.
 
-you can now leave `wpa_cli` by typing `quit`.
+For enterprise networks, for example *eduroam*, instead use the more
+complete `sudo nmtui` interface to configure the Wifi.
```

Comment:
Wifi -> wifi? Or go all-in with `Wi-Fi`?

Reply:
Fixed & force-pushed